snapcraft.yaml

```diff
@@ -8,10 +8,10 @@ description: |
   power supplies, ModemManager is able to prepare and configure the modems and
   setup connections with them.
   Please find the source code at
-  https://code.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git/modem-manager/+ref/snap-22
+  https://code.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git/modem-manager/+ref/snap-24
```

Comment:
We are using GitHub now (we should have changed this before)

Reply:
Updated, thank you for catching this!